### ISSUE TITLE
Random disconnectings in 7.31d

### ISSUE THREAD

**Issue by reacheight** [jeffhill/Dota2#1081]
### Description

Me and my friends experienced random disconnects after the patch. When it happens I can't reconnect without restarting Dota.

### Example Match ID (and possibly Timestamp)

6611395653 - 19:50

**Comment by eerieness** [comment]
Experiencing the same thing since the patch came out

**Comment by LostTeg** [comment]
Same. Some people can't load into a match and get disconnected in almost every game

**Comment by Qustio** [comment]
I'm assuming only the CIS region has connection issues. Have players from other countries/regions encountered this problem?

**Comment by eerieness** [comment]
> I'm assuming only the CIS region has connection issues. Have players from other countries/regions encountered this problem?

Yeah, i've thought about that. But why does it happen, even if that statement is true?

**Comment by jeffhill** [comment]
We think this was caused by a malfunctioning routing box in the EUE datacenter, which would cause problems regardless of which region you play on so long as you were connecting from the CIS region generally. We're removed that machine from the active routing pool until it's repaired.

Are folks still seeing this behavior?

**Comment by eerieness** [comment]
> We think this was caused by a malfunctioning routing box in the EUE datacenter, which would cause problems regardless of which region you play on so long as you were connecting from the CIS region generally. We're removed that machine from the active routing pool until it's repaired.
> 
> Are folks still seeing this behavior?

Hello. From now on, it seems that this problem is solved. I wasn't getting any disconnects for about 2 days now. Thanks for the response.
